Casks/font/font-t/font-tengwar-telcontar.rb

```diff
@@ -7,9 +7,8 @@
   homepage "https://freetengwar.sourceforge.net/tengtelc.html"
 
   livecheck do
-    url :homepage
-    regex(/href=.*?TengwarTelcontar[._-]v?(\d+)\.zip/i)
-    strategy :page_match
+    url "https://sourceforge.net/projects/freetengwar/rss?path=/TengwarFont"
+    regex(%r{url=.*?/TengwarTelcontar[._-]v?(\d+(?:\.\d+)*)\.zip}i)
```

Comment:
```suggestion
    regex(%r{url=.*?/TengwarTelcontar[._-]?v?(\d+(?:\.\d+)*)\.zip}i)
```

Considering that upstream uses a delimiter for some files and not for others, we may as well make this one optional to match the other.